app/assets/stylesheets/styles.scss

```diff
@@ -1,47 +1,59 @@
 $foundation-black: #222222;
 $bandtree-green: #43ac6a;
+$white: #ffffff;
+
+h1,h2,h3 {
+  font-weight: lighter;
+}
+
+h1 a {
+  color: $foundation-black;
+}
+
+h1 a:hover {
+  color: $bandtree-green;
+}
 
 .splash {
-  background-image: image-url(treesplash6);
+  background-image: image-url(front1);
   background-size: cover;
-  color: #ffffff;
+  height: 900px;
   position: relative;
-  height: 810px;
-  text-align: center;
-  text-shadow: 4px 2px 8px $foundation-black;
 }
 
-.splash a {
-  color: $bandtree-green;
+.splash-text {
+  color: $white;
+  text-align: center;
+  text-shadow: 4px 2px 8px $foundation-black;
+  a {
+    color: $bandtree-green;
+  }
 }
 
 .splash-title {
-  font-size: 20vw;
+  font-size: 18vw;
   font-weight: 900;
-  text-align: center;
+  margin-top: 75px;
 }
 
-.splash-text {
+.splash-intro {
   background-color: $foundation-black;
   font-size: 2em;
-  line-height: 36px;
-  margin: 0 -15px;
-  opacity: 0.95;
+  line-height: 40px;
+  margin: 0 -30px 0 -30px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 -30px` instead of `0 -30px 0 -30px`